src/splitInteger.test.js

```diff
@@ -8,12 +8,22 @@ describe(`Function 'splitInteger':`, () => {
   });
 
   it(`should return an array`, () => {
-
```

Comment:
I think, you shouldn't write values after splitIntegers, its work without it: splitInteger().toBe....

Reply:
You can use any option, with or without values